.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+yarn.lock
```

Comment:
I am not sure what is your setup here and how are you exporting this repo.

Reply:
We are not using `yarn.lock` in library packages.